packages/ui/src/documents/__tests__/CreateDocument.test.tsx

```diff
@@ -31,29 +31,6 @@ describe('Create Document', () => {
     });
   });
 
-  it('Should render the preloader', async () => {
```

Comment:
Do we replace this removed test case by some other coverage?

Reply:
there is no more preloader on Create Document so that's why I have removed this test